src/pages/iou/NewDistanceRequestPage.js

```diff
@@ -44,6 +46,7 @@ const defaultProps = {
 // You can't use Onyx props in the withOnyx mapping, so we need to set up and access the transactionID here, and then pass it down so that DistanceRequest can subscribe to the transaction.
 function NewDistanceRequestPage({iou, report, route}) {
     const iouType = lodashGet(route, 'params.iouType', 'request');
+    const isEditing = Navigation.getActiveRoute().includes('address');
```

Comment:
NAB: A comment here would be good. Also maybe make the name match the prop name.